tests/queries/0_stateless/01781_general_format_for_parser.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+TEST_OUTPUT=`$CLICKHOUSE_LOCAL --query "select toUInt8(1) as a, toUInt16(2) as b, '\xDE\xAD\xBE\xEF' as c, toFixedString('\xCA\xFE\xBA\xBE',4) as d from numbers(5) format Native" | md5sum | cut -d ' ' -f1`
+
+EXPECTED_OUTPUT=`echo 'd7255e6863ef058aa60064fab921fb2e'`
```

Comment:
Actually i've meant to check it via Kafka :) Smth similar to 
https://github.com/ClickHouse/ClickHouse/blob/fd5907a3328394d0c3ef5514026acc5d27221f48/tests/integration/test_storage_kafka/test.py#L484-L487

Reply:
> correctly report the error if the Native stream is broken in the middle)

I got it.